lib/rest.js

```diff
@@ -52,36 +50,36 @@ module.exports = class Rest {
         let page = 1;
         const baseUrl = url.split('?')[0];
         const isTransactionalMessageApi = this.isTransactionalMessageApi(baseUrl);
-        const queryParams = new URLSearchParams(url.split('?')[1]);
+        const queryParameters = new URLSearchParams(url.split('?')[1]);
         let collector;
         let shouldPaginate = false;
-        queryParams.set('$pageSize', Number(pageSize || 50).toString());
+        queryParameters.set('$pageSize', Number(pageSize || 50).toString());
         do {
-            queryParams.set('$page', Number(page).toString());
-            const temp = await this._apiRequest(
+            queryParameters.set('$page', Number(page).toString());
+            const temporary = await this._apiRequest(
```

Comment:
if we are renaming it - wouldn't `reponseBatch` or sth similar be more fitting?

Reply:
good idea :)